### PR TITLE
Feat: Add a solana developement complete guide for Solana Scribes

### PR DESCRIPTION
**Pull request by zerefwth**
### Description

---

**Title:** Solana Development Guide for Scribes Hackathon

**Description:**

This PR adds a concise Solana development guide for the Solana Scribes Hackathon participants. It covers setup, smart contract development with Rust and Anchor and frontend integration. Aimed at accelerating the learning curve for both new and seasoned developers, this guide is a stepping stone to innovative Solana-based projects.

**Highlights:**

- Quick setup instructions
- Smart contract development and deployment
- Frontend integration tips using Next.js
- Phantom integration

**Purpose:**

To equip future builders with the knowledge needed to build on Solana efficiently.

**Call to Action:**

Feedback and contributions are welcome to refine this guide further.

---

### CODE REVIEW

**Comment by nickfrosty** [comment]
Thanks for writing up this developer guide during the Scribes hackathon! It is a great start but needs some work to it. It has a bunch of extra paragraphs at the top that I don't think are needed. It was also hard to follow, at least partially because of the formatting and typos throughout. 

Unfortunately, since we can only publish one "Solana full stack development" guide on the Solana website and it will be a different one. So I will close this PR.

I would love for you to improve your guide and publish it to one of the popular publishing platforms like Medium, dev.to, or a personal blog.

**Comment by zerefwth** [comment]
@nickfrosty 
Thank you for your feedback. I've made the necessary improvements and published the guide on [dev.to](https://dev.to/zeref101/the-complete-guide-to-full-stack-solana-development-with-nextjs-anchor-and-phantom-4180) While I understand the decision to close the PR, I'm still eager to contribute. Could you confirm if I'm still eligible for the Scribes hackathon 2nd or 3rd prize tracks? Thank you.
